pkg/controllers/management/clusterprovisioner/provisioner.go

```diff
@@ -397,10 +425,12 @@ func (p *Provisioner) reconcileCluster(cluster *v3.Cluster, create bool) (*v3.Cl
 	} else {
 		logrus.Infof("Updating cluster [%s]", cluster.Name)
 
-		cluster, err = p.setClusterStatusUpdating(cluster)
+		v3.ClusterConditionUpdated.Unknown(cluster)
```

Comment:
can you add a comment here explaining why we do this. Just something simple, 1 line